tasks/main.yml

```diff
@@ -7,15 +7,23 @@
 
 - name: Create configuration directories
```

Comment:
- name:  Create TLS configuration directory

Reply:
Sorry, I just saw this too late. Do you mind opening a PR for this?